packages/SwingSet/src/kernel/state/kernelKeeper.js

```diff
@@ -507,6 +507,15 @@ export default function makeKernelKeeper(storage) {
     });
   }
 
+  function vatStats(vatID) {
+    return harden({
+      nextObjectId: storage.get(`${vatID}.o.nextID`),
```

Comment:
do these want to be `Nat(Number(xx))` instead of strings?

Reply:
good idea.